src/app/destiny1/loadout-drawer/LoadoutDrawerContents.tsx

```diff
@@ -145,6 +140,8 @@ async function pickLoadoutItem(
 ) {
   const loadoutClassType = loadout?.classType;
   function loadoutHasItem(item: DimItem) {
+    // TODO: Accessing id is not safe because we resolve emblems.
+    // Should we use the definitions here too?
```

Comment:
This is D1 stuff and only relevant for the "pick loadout item" logic, so maybe not that important?